linera-views/src/dynamo_db.rs

```diff
@@ -623,6 +623,13 @@ impl DynamoDbClientInternal {
         Ok(())
     }
 
+    async fn list_tables(
+        store_config: DynamoDbKvStoreConfig,
+    ) -> Result<Vec<String>, DynamoDbContextError> {
```

Comment:
This function and the similar ones would be better as methods of `DynamoDbKvStoreConfig`.

Reply:
(This would have solved the name collision)